massa-execution-worker/src/interface_impl.rs

```diff
@@ -143,6 +143,29 @@ impl InterfaceClone for InterfaceImpl {
     }
 }
 
+/// Helper function that creates an amount from a NativeAmount
+fn from_native_amount(amount: &NativeAmount) -> Result<Amount> {
```

Comment:
let's call it amount_from_native_amount :)

Reply:
done